metadata.yaml

```diff
@@ -14,6 +14,8 @@ maintainers:
   - https://launchpad.net/~canonical-is-devops
 assumes:
   - k8s-api
+  - juju >= 3.3
```

Comment:
Curious, why do we need `juju >= 3.3`?

Reply:
Since we're support noble as a run-on option, we'd need juju to be at least 3.3. On juju < 3.3 you'll get an error when specifying --base=ubuntu@24.04

Reply:
I'm wondering if we should set juju >= 3.1 since PS6 is not yet on 3.3 I believe? (I'm not so sure on this fact though) @Thanhphan1147